src/muparserx.mk

```diff
@@ -0,0 +1,25 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := muparserx
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := v3_0_3
+$(PKG)_CHECKSUM := d476899a024b6b720591484f615464dc1eb25b23
+$(PKG)_SUBDIR   :=
+$(PKG)_FILE     := $(PKG)_v3_0_2.zip
+$(PKG)_URL      := http://muparserx.googlecode.com/svn/archives/$($(PKG)_FILE)
+$(PKG)_DEPS     := gcc
+
+define $(PKG)_UPDATE
+echo "TODO: write update for muparserx"
+endef
+
+define $(PKG)_BUILD
+    mkdir '$(1)$(PKG)_$($(PKG)_VERSION).build'
```

Comment:
here and below, I'd rather put a `/` between `$(1)` and `$(PKG)_$($(PKG)_VERSION)`